.gitignore

```diff
@@ -1,7 +1,8 @@
 /doc
 /ebin
-/lib/*/ebin/*
-/lib/*/tmp
+/lib/**/ebin
```

Comment:
why you think they are not.
It solves the problem of interrupted tests and residual folders created in fixtures, that are deeper than one dir under lib/